eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -164,7 +168,10 @@ def params(self):
             offset = (int)(random.random() * math.fabs(t2 - t1))
 
             ts_max_ms = int(offset + min(t1, t2))
-        
+
+        if self._window_length == "random":
+            self._window_duration_ms = random.randrange(60*1000, 86400*100*1000, 60*1000)
```

Comment:
Any thoughts on what value to use as the maximum value for _window_duration_ms.? Does ts_max_ms makes sense? But then most of the values I get seem to be quite large..

Reply:
talk to Dimitrios offline and we probably want to use ts_max_ms, I will test with a a larger data set and see if i see the issue with window duration still seems to be weighted towards large values or if it was due to me running my test for a short period and with not a lot of data indexed.